catalog/spatial/csw/spatial-csw-source-common/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/common/source/CswFilterDelegate.java

```diff
@@ -130,104 +130,74 @@ public FilterType propertyIsEqualTo(String propertyName, String literal,
             boolean isCaseSensitive) {
         isComparisonOperationSupported(ComparisonOperatorType.EQUAL_TO);
         propertyName = mapPropertyName(propertyName);
-        if (isPropertyQueryable(propertyName)) {
```

Comment:
What if we ma dent his method abstract in the AbstractCswFilterDelegate, and then override it to return true for this implementation?